espnet2/asr/frontend/s3prl_frontend.py

```diff
@@ -0,0 +1,120 @@
+from argparse import Namespace
```

Comment:
The name of this file could be just `s3prl` because it is already inside the frontend folder.

Reply:
Thanks! Will fix it!